client/components/info-popover/index.jsx

```diff
@@ -11,84 +11,85 @@ import Popover from 'components/popover';
 import classNames from 'classnames';
 import analytics from 'lib/analytics';
 
-export default React.createClass( {
+export default class InfoPopover extends Component {
+	static propTypes = {
+		autoRtl: PropTypes.bool,
```

Comment:
Thanks for alphabetically sorting the props ❤️ 

Maybe also alphabetically sort the list of suppoted positions? :trollface: